core-java-modules/core-java-arrays-guides/src/test/java/com/baeldung/print2DArrays/Print2DArraysTest.java

```diff
@@ -0,0 +1,54 @@
+package com.baeldung.print2DArrays;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+
+import uk.org.webcompere.systemstubs.SystemOut;
+import uk.org.webcompere.systemstubs.annotations.SystemStub;
+
+public class Print2DArrayTest {
```

Comment:
The class name `Print2DArrayTest` does not match the file name `Print2DArraysTest.java`

Reply:
corrected

Reply:
There is still an issue here, if you look in the PR you will see two copies of the file:
https://github.com/eugenp/tutorials/pull/15281/files

Reply:
deleted redundant folder.